pkg/network/ndmz/dualstack.go

```diff
@@ -267,3 +284,21 @@ func waitIP6() error {
 	}
 	return nil
 }
+
+func stopBackgroundProbe() error {
+	probe, err := dhcp.NewBackgroundProbe(DMZPub4)
+	if err != nil {
+		return err
+	}
+
+	running, err := probe.IsRunning()
+	if err != nil {
+		return nil
```

Comment:
why ignoring the error? the error does not mean the service is running, no? This need to bubble up the stack